core/configuration/src/main/kotlin/features/RegisterFeatures.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 AndroidDev Discord Dev Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package features
+
+/**
+ * Initialises a number of features. This is a convenience function to avoid calling [Feature.init] repeatedly.
+ */
+suspend fun initFeatures(vararg features: Feature) {
+    features.forEach {
+        it.init()
+    }
```

Comment:
actually, this would be a good place to launch stuff in a supervisor scope and each feature could start in parallel and be initialized in parallel

Reply:
Done, how's it look?

Reply:
Looks good to me